packages/menu/src/MenuItem.tsx

```diff
@@ -125,6 +125,15 @@ const activeStyle = css`
 const disabledStyle = css`
   cursor: not-allowed;
   background-color: ${uiColors.gray.light3};
+
+  &:hover {
+    &:before {
```

Comment:
Small note: no need to nest these.

This should be identical, and more clear to read:
```scss
&:hover:before {
  background-color: unset;
}
```